src/app/_components/CheckBox/CheckBox.js

```diff
@@ -0,0 +1,172 @@
+'use client';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from '@/utils/cn';
+import { checkBoxVariants } from './styles';
```

Comment:
:green_circle: `checkBoxVariants` => `variants`, checkBox is already in folder name

Reply:
i've made it simillar to the buttonVariants pr. Since this is a named export, i think it's better to retain checkBoxVariants, in case it's needed somewhere else

Reply:
why would checkbox variants be needed somewhere else?